### PR TITLE
fixed comments on defines

### PR DESCRIPTION
**Pull request by fdarosa2663**
Sorry, was missing some info in the comments on the NO_BM defines, had a user contact me about it

### CODE REVIEW

File: props/saber_fett263_buttons.h [hunk 1]
```diff
@@ -251,10 +256,15 @@ SaberFett263Buttons() : PropBase() {}
 
       // EVENT_PUSH
       if (fabs(mss.x) < 3.0 &&
-          mss.y * mss.y + mss.z * mss.z > 120 &&
+          mss.y * mss.y + mss.z * mss.z > 100 &&
```
Comment (profezzorn) [inline props/saber_fett263_buttons.h:259]: these lines should not be here.
Please sync/pull/merge to resolve.

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h:259]: I don't understand, I changed it to 100 on last update, I found it was easier to replicate consistently

Comment (profezzorn) [reply props/saber_fett263_buttons.h:259]: And did you do a git pull with those changes before creating this PR?

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h:259]: yes, this was in the previous PR, this PR was just updates to the comments at the top of the file

Comment (profezzorn) [reply props/saber_fett263_buttons.h:259]: No. Your master branch is four commits behind mine.

Comment (fdarosa2663) [reply props/saber_fett263_buttons.h]: ah, sorry, ok I merged from master, it says no conflicts and and no commits behind

**Pull request merged**